airbyte-integrations/connectors/source-yahoo-finance-price/source_yahoo_finance_price/manifest.yaml

```diff
@@ -1,312 +1,73 @@
-version: 0.50.0
+version: 0.79.1
```

Comment:
Weird that this didn't get bumped to newer version

Reply:
I pruned Docker volume two days before, isn't this the latest version?